libzecale/circuits/aggregator_circuit.hpp

```diff
@@ -89,6 +86,10 @@ class aggregator_circuit
     std::shared_ptr<aggregator_gadget<wppT, nverifierT, NumProofs>>
         _aggregator_gadget;
 
+    /// Nested proof verification results packer
+    std::shared_ptr<libsnark::multipacking_gadget<libff::Fr<wppT>>>
```

Comment:
happy to stick to a `mutlipacking_gadget` but for now we seem to make the assumption that the results are packed in a single field element (hence, rely on a `packing_gadget` only)

Reply:
Thanks.  I forgot about that. `packing_gadget` makes life much easer.

Reply:
Yes, we can switch back to a multipacking gadget if we can support batches so big that the number of verified proofs is > bit-length of the underlying field elements, but I doubt we'll need that ;)